airbyte-integrations/connectors/source-braintree/source_braintree/schemas/transaction.py

```diff
@@ -61,7 +63,7 @@ class Transaction(BaseModel):
     created_at: datetime
     credit_card_details: CreditCard
     currency_iso_code: str
-    custom_fields: str
```

Comment:
why we delete custom_fields here?

Reply:
It's a custom object collection of field/value pairs, so it is dynamic - see here https://developer.paypal.com/braintree/docs/reference/response/transaction/python#custom_fields

But I made up for it by allowing additional fields here  - https://github.com/airbytehq/airbyte/pull/31489/files#diff-cce8c41640b4701bed6c37a533394d7a18f3636dc9cfbb401d4e7f87958db8b0R126